source/uwp/Renderer/lib/AdaptiveRichTextBlockRenderer.cpp

```diff
@@ -0,0 +1,44 @@
+#include "pch.h"
+
+#include "AdaptiveRichTextBlock.h"
+#include "AdaptiveRichTextBlockRenderer.h"
+#include "AdaptiveRenderContext.h"
+#include "Util.h"
+#include "XamlBuilder.h"
+#include "AdaptiveElementParserRegistration.h"
+
+using namespace Microsoft::WRL;
+using namespace Microsoft::WRL::Wrappers;
+using namespace ABI::AdaptiveNamespace;
+using namespace ABI::Windows::Foundation;
+
+namespace AdaptiveNamespace
+{
+    HRESULT AdaptiveRichTextBlockRenderer::RuntimeClassInitialize() noexcept try
+    {
+        return S_OK;
+    }
+    CATCH_RETURN;
+
+    HRESULT AdaptiveRichTextBlockRenderer::Render(_In_ IAdaptiveCardElement* cardElement,
+                                              _In_ IAdaptiveRenderContext* renderContext,
+                                              _In_ IAdaptiveRenderArgs* renderArgs,
+                                              _COM_Outptr_ ABI::Windows::UI::Xaml::IUIElement** result) noexcept try
```

Comment:
is this the spacing from clang?

Reply:
nope, fixed

---
In reply to: [266631607](https://github.com/Microsoft/AdaptiveCards/pull/2552#discussion_r266631607) [](ancestors = 266631607)